libs/crypto/crypto.hpp

```diff
@@ -63,8 +65,6 @@ namespace iroha {
    * @param seed
    * @return
    */
-  ed25519::keypair_t create_keypair(blob_t<32> seed);
-
-
+  keypair_t create_keypair(blob_t<32> seed);
```

Comment:
Maybe `create_keypair(void)` will be a better solution (no issues with seed)?

Reply:
We could create an overload for ```create_keypair(void)``` but do not remove a version with seed because we have 2 different ```create_seed()``` implementations now (and both are used). I can add it if you think it will be useful.

Reply:
Yep, don't need to be deleted and as long as pr related to reworking crypto imo this is fine to add